src/activity/components/createNoLudica/generalConfig/GeneralConfiguration.module.css

```
@@ -8,24 +9,24 @@
   gap: 1rem;
   margin-bottom: 2rem;
   padding-bottom: 1rem;
-  border-bottom: 2px solid #f3f4f6;
+  border-bottom: 2px solid var(--color-border);
 }
 
 .headerIcon {
   font-size: 2rem;
-  color: #667eea;
+  color: var(--color-primary);
```

En todas las demas actividades, el color aca no es --color-primary sinó --color-stat-4. Me gusta más el --color-primary, seguramente porque es el diferente jjajajaj pero bueno por ahora cambia este para que estén iguales a los otros.

```suggestion
  color: var(--color-stat-4);
```